demos/default/.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
can remove this if anyone feel strongly about not bumping node version for default demo - was part if initial exploration and was just left in here